README.md

```diff
@@ -1,10 +1,48 @@
-# default-SEM-model
+# model_seg_rat_axon-myelin_sem
+---
+
+## Model overview
+![image of segmentation obtained from this model](sem_model_preview.png)
+
 AxonDeepSeg default SEM model and testing image. This model works at a resolution of 0.1 micrometer per pixel and was trained on rat spinal cord data collected via a Scanning Electron Microscope (SEM).
 
+## Dependencies
+- [ivadomed](https://ivadomed.org/) commit: e6554281d07b5afef9e68ce8b02e86b02bd68363
+- [axondeepseg](https://axondeepseg.readthedocs.io/en/latest/) commit: 805868e39eddf620c9f3d60d313cadffb1121bfb
+
+## Clone this repository
```

Comment:
"Clone this repository" and "Get the data" are not necessary for segmentation because it is installed with ADS.

Maybe we could have 2 main sections:
1) Train and test model: (related to ivadomed)
2) Segment (related to ADS)

Reply:
I agree that this could baffle new users. I'll reorganize these sections.

Reply:
@mariehbourget I changed the structure to have 2 main sections like you suggested.

Reply:
Thanks, that looks great!